ocaml/xenopsd/xc/service.ml

```diff
@@ -619,15 +619,14 @@ module SystemdDaemonMgmt (D : DAEMONPIDPATH) = struct
     match (of_domid domid, is_running ~xs domid) with
```

Comment:
This (existing code) is makes some unnecessary calls, as `is_running` also calls `of_domid` internally. And now we don't even care about the result of `is_running` in the systemd case. So better would be to drop `is_running` from the match and call `Compat.is_running ~xs domid` in the `None` case below.